api.php

```diff
@@ -467,6 +466,29 @@ public function run($userID, $permissionIsExplicit) {
 	}
 }
 
+/**
+ * API entry game/setbackfromleft
+ */
+class SetBackFromLeft extends ApiEntry {
+	public function __construct() {
+		parent::__construct('game/setbackfromleft', 'JSON', '', array('gameID','countryID'));
+	}
+	public function run($userID, $permissionIsExplicit) {
+		global $Game, $DB;
+		error_log("000");
```

Comment:
Should this and the other error_logs below be here?

Reply:
Thanks for catching this.